tools/ci_build/github/azure-pipelines/linux-qnn-ci-pipeline.yml

```diff
@@ -25,6 +25,9 @@ jobs:
 
       - script: |
           env | egrep -e QNN
```

Comment:
Env may contains secrets(such as SAS keys, tokens). This command may expose them.

Reply:
I removed this step. Thanks.